.circleci/config.yml

```diff
@@ -182,6 +182,40 @@ jobs:
           at: ~/
       - run: ../../.circleci/test-js-client.sh
 
+  fxa-email-event-proxy:
+    docker:
+      - image: circleci/node:8
+    working_directory: ~/project/packages/fxa-email-event-proxy
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run: npm ci
+      - run: npm run lint
+      - run: npm t
+
+  fxa-email-service:
+    docker:
+      - image: circleci/node:10
+      - image: mysql:5.6
+        environment:
+          - MYSQL_DATABASE: fxa
+          - MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          - MYSQL_ROOT_PASSWORD: ''
+      - image: redis
+    working_directory: ~/project/packages/fxa-email-service
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run: |
+          curl https://sh.rustup.rs > rustup-init.sh
+          . ./rustup-init.sh -y
+          rm ./rustup-init.sh
+          . "$HOME/.cargo/env"
+          rustup component add rustfmt-preview
+          cargo fmt -- --check
+          cargo b
+          ./scripts/test_with_authdb.sh
```

Comment:
we could artifact the db-mysql log like we do in `test-content-server`. maybe it'll help debug the unicode issue